### PR TITLE
Feature/ssvc 459

### PR DESCRIPTION
**Pull request by pjc3320**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -543,5 +560,12 @@ Represents a system of record user.
...
 + `userName`: soruser123 (string, required) - The user name for the system of record user.
-+ `sourceAbbreviation`: SD (string, required) - The the source abbreviation for the system of record(like SD for School Dude or FD for Facility Dude).
++ `sourceAbbreviation`: SD (string, required) - The source abbreviation for the system of record(like SD for School Dude or FD for Facility Dude).
 + `userSummaryId`: 1 (number, required) - {INT} The dude platform user.
+
+## System Of Record ApiClient (Api Client Audit)
+Represents a mapping between an Api Client and a System of Record.
+
+### Properties
++ `apiClientIdentifier`: abc123 (string, required) - The client identifier for the system of record.
```
Comment (agilenut) [inline apiary.apib]: We used MyClientName in the method definition but abc123 in the data structure. We should probably change the data structure to match.

File: apiary.apib [hunk 2]
```diff
@@ -543,5 +560,12 @@ Represents a system of record user.
 
 ### Properties
 + `userName`: soruser123 (string, required) - The user name for the system of record user.
-+ `sourceAbbreviation`: SD (string, required) - The the source abbreviation for the system of record(like SD for School Dude or FD for Facility Dude).
++ `sourceAbbreviation`: SD (string, required) - The source abbreviation for the system of record(like SD for School Dude or FD for Facility Dude).
```
Comment (agilenut) [inline apiary.apib]: Might want to add a space between record and (

File: apiary.apib (hunk 1 above)
Comment (pjc3320) [inline apiary.apib]: @agilenut "MyClientName" is for the Dude API Client identifier; the "apiClientIdentifier" is the partnerId being returned to the caller.

**Comment by pjc3320** [comment]
Oops... merging to master instead of orange.

**Pull request closed without merging**